### PR TITLE
Added Tracker API.

### PR DESCRIPTION
**Pull request by jonnyarnold**
The Tracker API is a fast endpoint that collects event information. It uses the Gin library to build the API.

The information will be stored in DynamoDB in future; for now, it discards the information.

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
_If you're new, read the [guidelines](https://github.com/reevoo/guidelines/blob/master/pull_requests.md)._

Good code is **tested**, **easy to understand** and **able to cope when things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [x] Description explains what you have done and why.
- [x] Tests have been written (or you have explained why you don't need tests).
- [x] Documentation/comments have been added/updated.

**For the Reviewer**
- [x] Tests are green on CI (or pulled/tested on your machine).
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] The code is easy to understand.

File: main/server.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+var (
+	dynamoUri           = os.Getenv("DYNAMODB_URI")
+	env                 = os.Getenv("GO_ENV")
+	Term      os.Signal = syscall.SIGTERM
+)
+
+func main() {
+	go exitOnInterrupt()
+
+	// Release mode reduces the amount of logging.
```
Comment (errm) [inline main/server.go:16]: I think it mainly makes everything faster, in return for less nice backtraces . . .

File: errors.go [hunk 2]
```diff
@@ -1,37 +1,60 @@
...
+	packet := raven.NewPacket(err.Error(), raven.NewException(err, raven.NewStacktrace(2, 3, nil)))
+	raven.Capture(packet, err.ToMap())
+}
+
+// Logs errors to the console.
+type ConsoleLogger struct{}
+
+// Logs an error to the console.
+func (ConsoleLogger) LogError(err TrackerError) {
+	fmt.Printf("[ERROR] %s", err.Error())
```
Comment (errm) [inline errors.go]: Perhaps log the errors to stderr?

Might be useful if we wanted to make a backend that just logs the events to stdout then we could consume using another process . . . . (logstash or somesuch)

File: main/server.go [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+	server := NewServer(ServerParams{
+		EventStore:  DynamoDBEventStore{},
+		ErrorLogger: SentryErrorLogger{},
+	})
+
+	server.Run(":3000")
+}
+
+func exitOnInterrupt() {
```
Comment (errm) [inline main/server.go]: Why do we need this?

Comment (jonnyarnold) [inline main/server.go]: Oops - I stole it from [your other code](https://github.com/reevoo/mark_tracking_go/blob/master/mark_tracking_go/server.go#L42-L48), but that's there so the queue is finished. Removed!

File: errors.go (hunk 2 above)
Comment (jonnyarnold) [inline errors.go]: Good point. Fixed.

**Comment by reevoo-samuel** [comment]
I don't see any comments on your Pull Request.

Are you too good for code reviews now?
No. No you aren't.

Maybe you forgot to say who reviewed it. That's fine, but let me know.

**Pull request merged**